index.bs

```diff
@@ -2339,7 +2339,7 @@ optionally evidence of [=user consent=] to a specific transaction.
     This dictionary is used to supply additional parameters when creating a new credential.
 
     :   <dfn>type</dfn>
-    ::  This member specifies the type of credential to be created.
+    ::  This member specifies the type of credential to be created. The value SHOULD be a member of {{PublicKeyCredentialType}} but [=client platforms=] MUST ignore unknown values, treating an unknown value as if the member was unset.
```

Comment:
Since this is a required member, treating it “as if the member was unset” would mean a parse error, which is what we're trying to avoid. How about “ignoring any [[PublicKeyCredentialParameters]] with an unknown type”.

Reply:
Agreed, thanks.